content/blog/posts/texas-attorney-general’s-sites-for-death-and-shooting-reports-go-dark.md

```diff
@@ -0,0 +1,32 @@
+
+Every week, law enforcement agencies file dozens of reports with the Attorney General’s office that provide a glimpse into incidents like an officer’s use of deadly force, an incarcerated individual’s death, and shootings in which an officer is shot or shoots a civilian, resulting in an injury or a death. The office then posts these reports online – except that hasn't happened since severe winter weather hit the state and all updates ceased.  
+
+On Feb. 15, I noticed that both repositories were empty. After a couple of weeks, some reports appeared, but both collections are still missing all reports filed in 2021 and nearly all the ones that had been filed in 2020, as well. 
+
+State laws require agencies to file these reports with the Texas Attorney General within 30 days of an incident in which an [officer is shot or shoots someone](https://statutes.capitol.texas.gov/Docs/CR/htm/CR.2.htm#2.139), or when [someone dies in their custody](https://statutes.capitol.texas.gov/Docs/CR/htm/CR.49.htm). The office then posts these reports online – a move stipulated by statute for the officer-involved shooting reports. 
+
+Said another way: By not publishing the reports on officer-involved shootings, the OAG is violating state law. 
+
+Back when the sites went down, I sent an email informing the office as such, as I have done in the past. On Feb. 22, I heard back:
+
+> Hi Ms. Moravec,
+>
+> The OAG was closed all last week due to the weather and the electrical outages.  I have reported the situation to IT this morning and they let me know they are working on it.
+>
+> As soon as I hear back from them, I will let you know.  Thank you for reporting this to us.
+
+IT then followed up that “Salesforce recently rolled out another update,” which caused the outage. About a week later, many of the cases had been restored, but hundreds from this year and last year remain missing, which I again reported to the Attorney General’s office. 
+
+This outage means that when someone dies in law enforcement custody – such as Marvin Scott III, 26, who died in the Collin County Jail in March, the custodial death report that officials must file related to the death isn't visible online. That being said, reports (once they are filed) and data are still obtainable through open records requests. 
+
+While the sites have been malfunctioning, TJI has continued to obtain data on these incidents from the Attorney General’s office through our standard monthly open records requests. While the data on our site reflects reports filed through March 1, the actual reports are still unavailable to the public. 
```

Comment:
```suggestion
While the sites were malfunctioning, TJI continued to obtain data on these incidents from the Attorney General’s office through our standard monthly open records requests. We also stayed in touch with the office, which informed us March 29 that the sites had been fully restored. 
```

Reply:
Do we know whether the OAG website has any records missing (during the outage) at this point or whether they fully fixed the problem?